incubator/elasticsearch/values-production.yaml

```diff
@@ -0,0 +1,211 @@
+## Bitnami Elasticsearch image version
+## ref: https://hub.docker.com/r/bitnami/elasticsearch/tags/
+##
+image:
+  registry: docker.io
+  repository: bitnami/elasticsearch
+  tag: 6.2.3-r3
```

Comment:
use the rolling tag instead (6.2.3)

Reply:
thanks for stepping in here. I missed it :slightly_smiling_face:

Reply:
👍